lib/mi_bridges/driver/job_income_information_page.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class JobIncomeInformationPage < BasePage
+      delegate :primary_member, to: :snap_application
+
+      delegate(
+        :employment_status,
+        :first_name,
+        to: :primary_member,
+      )
+
+      def setup; end
+
+      def fill_in_required_fields
+        check_fulltime_employment_status
+        check_self_employment_status
+        check_fap_program_benefits
+      end
+
+      def continue
+        click_on "Next"
+      end
+
+      private
+
+      def check_fulltime_employment_status
```

Comment:
this method name brings up an interesting point: we ask if someone is employed, not specifically if employed FT. (not a problem for this PR, just something to think about for the app)